internal/app/networkd/pkg/nic/vars.go

```diff
@@ -0,0 +1,193 @@
+		"num peer notif", "all slaves active", "min links", "lp interval", "packets per slave",
+		"ad lacp rate", "ad select", "ad innfo", "ad actor sys prio", "ad user port key",
+		"ad actor system", "tlb dynamic lb", "peer notif delay",
+	}[int(b)]
+}
+
+// https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/if_bonding.h
+type BondMode byte
+
+const (
+	BOND_MODE_ROUNDROBIN BondMode = iota
+	BOND_MODE_ACTIVEBACKUP
+	BOND_MODE_XOR
+	BOND_MODE_BROADCAST
+	BOND_MODE_8023AD
+	BOND_MODE_TLB
+	BOND_MODE_ALB
+)
+
+func (b BondMode) String() string {
+	return [...]string{"balance-rr", "active-backup", "balance-xor", "broadcast", "802.3ad", "balance-tlb", "balance-alb"}[int(b)]
+}
+
+func BondModeByName(mode string) (bm BondMode, err error) {
+	switch mode {
+	case "balance-rr":
+		bm = BOND_MODE_ROUNDROBIN
+	case "active-backup":
+		bm = BOND_MODE_ACTIVEBACKUP
+	case "balance-xor":
+		bm = BOND_MODE_XOR
+	case "broadcast":
+		bm = BOND_MODE_BROADCAST
+	case "802.3ad":
+		bm = BOND_MODE_8023AD
+	case "balance-tlb":
+		bm = BOND_MODE_TLB
+	case "balance-alb":
+		bm = BOND_MODE_ALB
+	default:
+		err = fmt.Errorf("invalid bond type %s", mode)
+	}
+
+	return bm, err
+}
+
+type BondXmitHashPolicy byte
+
+const (
+	BOND_XMIT_POLICY_LAYER2 BondXmitHashPolicy = iota
+	BOND_XMIT_POLICY_LAYER34
+	BOND_XMIT_POLICY_LAYER23
+	BOND_XMIT_POLICY_ENCAP23
+	BOND_XMIT_POLICY_ENCAP34
+)
+
+func (b BondXmitHashPolicy) String() string {
+	return [...]string{"layer2", "layer3+4", "layer2+3", "encap2+3", "encap3+4"}[int(b)]
+}
+
+func BondXmitHashPolicyByName(policy string) (xmit BondXmitHashPolicy, err error) {
+	switch policy {
+	case "layer2":
+		xmit = BOND_XMIT_POLICY_LAYER2
+	case "layer3+4":
+		xmit = BOND_XMIT_POLICY_LAYER34
+	case "layer2+3":
+		xmit = BOND_XMIT_POLICY_LAYER23
+	case "encap2+3":
+		xmit = BOND_XMIT_POLICY_ENCAP23
+	case "encap3+4":
+		xmit = BOND_XMIT_POLICY_ENCAP34
+	default:
+		err = fmt.Errorf("invalid xmit hash policy %s", xmit)
+	}
+	return xmit, nil
+}
+
+type LACPRate int
+
+const (
+	LACP_RATE_SLOW LACPRate = iota
+	LACP_RATE_FAST
+)
+
+func (l LACPRate) String() string {
+	return [...]string{"slow", "fast"}[l]
+}
+
+func LACPRateByName(mode string) (rate LACPRate, err error) {
+	switch mode {
+	case "slow":
+		rate = LACP_RATE_SLOW
+	case "fast":
+		rate = LACP_RATE_FAST
+	default:
+		err = fmt.Errorf("invalid lacp rate %s", mode)
+	}
+	return rate, err
+}
+
+// TODO:
+/*
+ static const char *lacp_rate_tbl[] = {
+ 	"slow",
+ 	"fast",
+ 	NULL,
+ };
+
+ static const char *ad_select_tbl[] = {
+ 	"stable",
+ 	"bandwidth",
+ 	"count",
+ 	NULL,
+ };
+
+ static const char *arp_validate_tbl[] = {
+ 	"none",
+ 	"active",
+ 	"backup",
+ 	"all",
+ 	NULL,
+ };
+
+ static const char *arp_all_targets_tbl[] = {
+ 	"any",
+ 	"all",
+ 	NULL,
+ };
+
+ static const char *primary_reselect_tbl[] = {
+ 	"always",
+ 	"better",
+ 	"failure",
+ 	NULL,
+ };
+
+ static const char *fail_over_mac_tbl[] = {
+ 	"none",
+ 	"active",
+ 	"follow",
+ 	NULL,
+ };
+*/
```

Comment:
nit: I'd prefer not to have commented C code here. Is there another way we can remind ourselves of this?

Reply:
Maybe a ticket is a better way to log this? We can remove from here and have the ticket in the backlog. More likely to get lost in the code.